docs/_data/table/en/ReadType.yml

```diff
@@ -4,6 +4,6 @@ CACHE_PROMOTE:
   for each completely read data block. This is the default read type.
 CACHE:
   If the data was not in the Alluxio storage of the local worker, a replica will be added to the
-  local Alluxio worker for each completely read data block.
```

Comment:
@jsimsa Is the ticket referring to how `alluxio.user.file.cache.partially.read.block` determines whether the block needs to be read completely? I think it's better to mention the config value in the docs so that it's clear how `CACHE` behaves with it and without it.